internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -8,6 +8,23 @@ admin:
     socket_address:
       address: {{ .AdminServer.Address }}
       port_value: {{ .AdminServer.Port }}
+{{- if .EnableStatConfig }}
```

Comment:
do we need an extra bool field, can we use ` {{- if not (kindIs "invalid" .StatsMatcher) }}`
relates to https://github.com/Masterminds/sprig/issues/53

Reply:
get it